src/components/CirclesHeaderSection/CirclesHeaderSection.tsx

```diff
@@ -6,7 +6,7 @@ import { makeStyles, Theme } from '@material-ui/core';
 
 import { CircleButton } from 'components';
 import { useApiBase } from 'hooks';
-import { useSelectedCircle, useCircles } from 'recoilState';
+import { useSelectedCircle, useCircles } from 'recoilState/app';
```

Comment:
Undo this, I should probably delete index, @levity asked me to import from the individual files